subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio_nrf5.h

```diff
@@ -40,21 +40,22 @@
 #include <hal/nrf_vreqctrl.h>
 #include "radio_nrf5340.h"
 #elif defined(CONFIG_SOC_SERIES_NRF54LX)
+#include <hal/nrf_ppib.h>
```

Comment:
Should it be put instead under a `CONFIG_SOC_COMPATIBLE_NRF54LX` ?
and or the MDK `PPIB_PRESENT` macro?

Reply:
I meant as a separate ifdef from this ones adding the radio headers.